WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/ProductFormViewController+ProductImageUploaderTests.swift

```diff
@@ -0,0 +1,101 @@
+import Combine
+import XCTest
+
+@testable import WooCommerce
+
+final class ProductFormViewController_ProductImageUploaderTests: XCTestCase {
+    private let window = UIWindow(frame: UIScreen.main.bounds)
+
+    override func setUp() {
+        super.setUp()
+        window.makeKeyAndVisible()
+    }
+
+    override func tearDown() {
+        window.resignKey()
+        window.rootViewController = nil
+
+        super.tearDown()
+    }
+
+    func test_triggering_viewDidLoad_invokes_stopEmittingStatusUpdates() throws {
```

Comment:
Should we rename these methods as well? i.e. `test_triggering_viewDidLoad_invokes_stopEmittingErrors`